Support to resources exposed as text/csv

haroldo-bonette opened the pull request:
Some Salesforce resources like sobjects/EventLogFile/[id]/LogFile are exposed as base64 (csv) format.
This change decodes the byte format into string and converts it to json, the json string is then parsed using json.loads which returns an OrderedDict

jon-wobken commented:
Pandas is such a large library I am hesitant to force it as a requirement and if this is a needed functionality either handle it they a different means or using a smaller package.

haroldo-bonette commented:
> Pandas is such a large library I am hesitant to force it as a requirement and if this is a needed functionality either handle it they a different means or using a smaller package.

@jon-wobken I have refactored it and removed Pandas
